test/spec/api/create-vis.spec.js

```diff
@@ -242,21 +242,21 @@ describe('src/api/create-vis', function () {
     it('should instantiate map', function (done) {
       this.vis = createVis('domId', fakeVizJSON, {});
 
-      _.defer(function () {
+      setTimeout(function () {
         expect(mapInstantiationRequestDone()).toEqual(true);
         done();
-      });
+      }, 25);
```

Comment:
I tried to avoid this timeout but we're using `_.debounce` in src/geo/map.js and jasmine.clock didn't work :(